src/content/articles/pygotham-day-II.md

```
@@ -14,7 +14,7 @@ description: Second Day of my first conference ever. I tried to make good notes
 
 Day 2 of the PyGotham conference was on Saturday. I expected there to be way more people, but surprisingly, the opposite has happened. Day 2 count was twice as small, or at least that is how it felt. Below are the note from Day 2.
 
-You can see Day 1 notes [here](pygotham-2019-day-one)
+You can see Day 1 notes [here](/pygotham-day-one)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)